web/ui/react-app/src/constants/constants.tsx

```diff
@@ -0,0 +1 @@
+export const APIPATH = '../api/v1';
```

Comment:
We'd use snake case for upper-cased constants:

```suggestion
export const API_PATH = '../api/v1';
```